docs/documentation/writing-documentation.md

```diff
@@ -114,7 +114,8 @@ To contribute content to the OSG, follow these steps to submit a pull request wi
                If `mkdocs.yml` contains does not contain a `nav:` section,
                add the above to the `pages:` section instead.
                This means that the repository is using an older version of MkDocs
-               and will need to be [upgraded to MkDocs v1](#upgrading-to-mkdocs-v1)
+               and will need to be
+               [transitioined to GitHub Actions](publish-osg-pages.md#transitioning-to-github-actions).
```

Comment:
```suggestion
               [transitioned to GitHub Actions](publish-osg-pages.md#transitioning-to-github-actions).
```

Reply:
Man my 'i' key has been causing all sorts of grief lately.